src/pages/candidate-details/__tests__/candidate-details.spec.ts

```diff
@@ -17,7 +18,7 @@ describe('CandidateDetailsPage', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [CandidateDetailsPage],
-      imports: [IonicModule, AppModule],
+      imports: [IonicModule, AppModule, ComponentsModule],
```

Comment:
Instead of adding the entire Components Module could we not just use this in the declarations:

`MockComponent(DisplayAddressComponent),`

Reply:
Good shout